README.md

```diff
@@ -1,13 +1,43 @@
-# Send VPC Flow logs to Splunk via Kinesis Firehose
+# Forward VPC Flow logs to Splunk via AWS Firehose
 
-This module configures a Kinesis Firehose, sets up a subscription for a desired CloudWatch Log Group to the Firehose, and sends the log data to Splunk.
+This module configures a Kinesis Firehose, sets up a subscription for a desired 
+CloudWatch Log Group to the Firehose, and sends the log data to Splunk.
 
-According to [AWS Big Data Blog](https://aws.amazon.com/blogs/big-data/ingest-vpc-flow-logs-into-splunk-using-amazon-kinesis-data-firehose/), a Lambda is no longer required to transform the VPC flow logs according to the 7.3.0 version in Splunk AWS Add-on, hence this module does not contain any Lambda processing any more. If you need to send data to a Splunk system with an older Add-on, please take a look to v0.1.0 of this module.
+# Prerequisites
+
+* Terraform >= 1.7.0
```

Comment:
@rporres I actually though about bumping the version in `terraform.tf`.
I mean, how do we know it works with Terraform v1.0.0? 
I would rather set the bar higher and support only one previous X version. 
WDYT?

Reply:
I know it works with 1.1 and 1.4. I started with 1.1 and I currently make it work with 1.4.7, which is the version that we have most of the stuff in our infra repo.

Since terraform license has changed in 1.5 in a dramatic way, I don't want to put a restriction there.

The terraform binary itself version is not that important, it is the provider version, and we're forcing to be in the newest branch.

Reply:
@rporres Valid points.
I will change it to >=1.0.0

I wonder it works with [OpenTofu](https://opentofu.org) :)

Reply:
I guess it should work :)

Reply:
@rporres Changed to 1.0.0